mysql/toddl.go

```diff
@@ -214,6 +215,17 @@ func cvtForeignKeys(conv *internal.Conv, srcTable string, srcKeys []schema.Forei
 				conv.Unexpected(fmt.Sprintf("Can't map foreign key for table: %s, referenced table: %s, column: %s", srcTable, key.ReferTable, col))
 				continue
 			}
+			if _, found := conv.SpSchema[spReferTable].ColDefs[spReferCol]; !found {
+				// Spanner foreign key referenced columns are case sensitive, so
+				// if we don't find column with direct look up, we compare referenced
+				// column with all columns with lower case in referenced table.
+				for _, refCol := range conv.SpSchema[spReferTable].ColNames {
+					if strings.ToLower(refCol) == strings.ToLower(spReferCol) {
+						spReferCol = refCol
+						break
```

Comment:
Without looking too deeply into whether it's a possible scenario given err1 and err2 above, I am wondering whether we should try to report conv.Unexpected error here if there are no matches found even with a ToLower?

Reply:
I see that this issue doesn't happen now in updated code. Resolving.